packages/loaders/src/tiff/index.ts

```diff
@@ -96,59 +97,102 @@ export async function loadOmeTiff(
   return images === 'all' ? loaders : loaders[0];
 }
 
+function getImageSelectionName(
+  imageName: string,
+  imageNumber: number,
+  imageSelections: [number, OmeTiffSelection][]
+) {
+  return imageSelections.length === 1
+    ? imageName
+    : imageName + `_${imageNumber.toString()}`;
+}
+
 /**
- * Opens multiple tiffs as a multidimensional "stack" of 2D planes. Returns the data source and OME-TIFF-like metadata.
- *
+ * Opens multiple tiffs as a multidimensional "stack" of 2D planes.
+ * Also supports loading multiple slickes of a stack from a stacked tiff.
+ * Returns the data source and OME-TIFF-like metadata.
  *
  * @example
  * const { data, metadata } = await loadMultiTiff([
  *  [{ c: 0, t: 0, z: 0 }, 'https://example.com/channel_0.tif'],
  *  [{ c: 1, t: 0, z: 0 }, 'https://example.com/channel_1.tif'],
- *  [{ c: 2, t: 0, z: 0 }, 'https://example.com/channel_1.tif'],
+ *  [[[0, { c: 2, t: 0, z: 0 }], [1, { c: 3, t: 0, z: 0 }]], 'https://example.com/channels_2-3.tif'],
```

Comment:
is the use-case here loading only certain images from a given tiff?  does this happen in practice, building stacks from sub-images of a given tiff, but ones that are out of order?

if so, I would want to think about this API a bit, not sure about this.  But if this is just anticipating generalizations without a concrete use-case at the moment, I'm not sure this is necessary.

Reply:
This is to load only certain images from a given tiff, although I can imagine other uses. Sometimes we'll have images where some of the sub-images are small regions of the full resolution images that are included for QC. I haven't tested loading images of different resolutions with Viv, but if Viv is able to load them without error I can remove this.

If I do remove this, what should the API look like instead? How should a user indicate that a stacked TIFF contains a C, Z, or T stack?

Reply:
Hmmmm, I don't know how that would work.  Maybe try it?  If it doesn't work, and it doesn't seem like that can be fixed then we'll need to either let this through or come up with something better.  Perhaps an array that is empty in the entries where there is no selection?  But not sure...

Reply:
Alright, Viv is not able to load multiple images with different resolutions. It loads all of the images, but then is not able to display any image whose resolution is different from that of the first image.

I liked your array idea better than what I was originally using. I updated the input selections to `OmeTiffSelection | (OmeTiffSelection | undefined)[] ` instead of the object with indexes.